src/components/Shared/TabbedContainer/index.ts

```diff
@@ -0,0 +1 @@
+export { TabbedContainer } from "./TabbedContainer";
```

Comment:
I think this component could probably live in "Layout" module.

Reply:
I agree. Good point

Reply:
Moved it to the Layout module.